x/ibc/testing/chain.go

```diff
@@ -388,7 +388,7 @@ func (chain *TestChain) AddTestConnection(clientID, counterpartyClientID string)
 // created given a clientID and counterparty clientID. The connection id
 // format: <chainID>-conn<index>
 func (chain *TestChain) ConstructNextTestConnection(clientID, counterpartyClientID string) *TestConnection {
-	connectionID := fmt.Sprintf("%s-%s%d", chain.ChainID, ConnectionIDPrefix, len(chain.Connections))
+	connectionID := fmt.Sprintf("%s%d", connectiontypes.ConnectionPrefix, len(chain.Connections))
```

Comment:
nit, I think the sep was ok here

Reply:
we cannot choose identifiers any longer, this must match the auto derived format

Reply:
I can make a helper function in connection/channel types. `FormatConnectionIdentifier`?

Reply:
cc @cwgoes should the identifier be `channel0` or `channel-0`?

Reply:
I prefer the latter

Reply:
I updated to use `-`. I don't see any reason not to